swarm/network/stream/syncer_test.go

```diff
@@ -38,7 +40,7 @@ import (
 	"github.com/ethereum/go-ethereum/swarm/testutil"
 )
 
-const dataChunkCount = 200
+const dataChunkCount = 1000
 
 func TestSyncerSimulation(t *testing.T) {
```

Comment:
@nonsense i think i'm going to just revert the change for this specific test (and the corresponding `testSyncBetweenNodes`), or maybe even remove it altogether with the new tests below. WDYT? the actual comparison on this test below is wrong - `if shouldUntil != until` can flake because the chunk will not necessarily be on the same bin index on the second node. i suggest we remove this test altogether.

Reply:
Sure, makes sense.

Reply:
done